### PR TITLE
Fix incorrect netmask sent to client with static IP set in RADIUS (Bug #5129)

### PR DESCRIPTION
**Pull request by doktornotor**
https://redmine.pfsense.org/issues/5129

### CODE REVIEW

File: src/etc/inc/openvpn.auth-user.php [hunk 1]
```diff
@@ -186,15 +186,13 @@ function getNasIP() {
...
+	if (isset($attributes['framed_mask'])) {
 		$content .= "topology subnet\n";
 		$content .= "ifconfig-push {$attributes['framed_ip']} {$attributes['framed_mask']}";
 	} else {
-*/
-	$content .= "topology net30\n";
-	$content .= "ifconfig-push {$attributes['framed_ip']} ". long2ip((ip2long($attributes['framed_ip']) + 1));
-//	}
+		$content .= "topology net30\n";
+		$content .= "ifconfig-push {$attributes['framed_ip']} ". long2ip((ip2long($attributes['framed_ip']) + 1));
```
Comment (jim-p) [inline src/etc/inc/openvpn.auth-user.php]: As long as this is being fixed, we may as well fix the consistency in client/server address as well. In client-specific config/overrides the server address is made one less than the client address, not one higher, which is consistent with OpenVPN's own internal address assignment.  See https://github.com/pfsense/pfsense/blob/master/src/etc/inc/openvpn.inc#L1053 - thoughts?

Should be as easy and changing that +1 to -1.

Comment (doktornotor) [inline src/etc/inc/openvpn.auth-user.php]: Yeah, definitely makes sense.

@ermal - any particular reason here why's it done the other way round for RADIUS?

Comment (jim-p) [inline src/etc/inc/openvpn.auth-user.php]: Most likely it was done at the request of the original customer that sponsored the work, and probably should have been flipped (or made an option) when imported. Given how little it is used the impact should be fairly low.

**Comment by doktornotor** [comment]
Yeah I don't think there's actually much to break here. :)

**Pull request merged**